client/src/components/MovieRec.jsx

```diff
@@ -5,23 +5,21 @@ import React from 'react'
 
 export default function MovieRec({ movies }) {
 
-  // const selectRandomMovie = (movies) => {
-  //   const randomIndex = Math.floor(Math.random() * movies.length);
-  //   return movies[randomIndex];
-  // }
+  const selectRandomMovie = (movies) => {
+    const randomIndex = Math.floor(Math.random() * movies.length);
+    return movies[randomIndex];
+  }
 
-  // const randomMovie = selectRandomMovie(movies);
+  const randomMovie = selectRandomMovie(movies);
```

Comment:
I think your rendering issues come from here. This should be moved to a React `useState` hook.